### ISSUE TITLE
Localization Issue: "Light" being shared for the theme and the encumbrance level

### ISSUE THREAD

**Issue by Frerol** [richardwilkes/gcs#685]
Basically, in English, "Light" can refer to both a color theme and the weight someone is bearing, aside from other possible uses, and it may not be so with other languages, such as it is with Brazilian Portuguese. If the localization file could differentiate between the two things specifically for the "Light" word, it would help localize the program better. Such an issue could happen to other terms, such as "Dark", but I didn't check them.

**Comment by richardwilkes** [comment]
Unfortunately, there isn't a good solution for this with the current code base. I don't personally use the localizations, so don't have a lot of incentive to do even more work here myself. Perhaps someone will come along who has the knowledge and motivation to work on the problem, though, so I'll leave it as an open issue.

**Comment by chuckn408** [comment]
Correct me if I misunderstood, but the request is to enable a cherry-pick localization of specific keywords.  The task would be more tedious and unnecessary than an entire translation.  For the latter, that could be found here::
https://github.com/richardwilkes/gcs_translations

**Comment by richardwilkes** [comment]
Yes, the task is to modify the i18n code to support having multiple variants for a specific blob of text that are keyed to something in the code such that the correct variant can be selected at runtime. That is problematic, as it requires me to know which words are reused in ways that have different meanings in languages I don't speak and then update the places where these are used to provide some sort of context key that can be selected from. Hence, why I marked this as "help wanted", because it isn't something I can actually resolve myself.

**Comment by richardwilkes** [comment]
I'm closing this, as I do not see a maintainable way to implement it. If you have a specific suggestion and code to back it up, I'm open to trying it. In that case, please open a PR illustrating the solution.
